django/pieskiUW/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+
+class Pet(models.Model):
+    """Model of pet table"""
+
+    SEX = (
+        ("m", "male"),
+        ("f", "female"),
+    )
+    no = models.CharField(max_length=7)
```

Comment:
Increase max_length to something more. We should use the exact value, because we don't know if there are bigger numbers. I propose length of 20.